pages/singlesite/[slug].js

```diff
@@ -11,45 +16,234 @@ export default function singleSite() {
   const currentSite = allSites.find((site) => site.slug === slug);
 
   if (!currentSite) {
-    return;
-    <>
-      <p>Hier gibt es leider nichts zu entdecken ...</p>
-    </>;
+    return (
+      <>
+        <p>Hier gibt es leider nichts zu entdecken ...</p>
+      </>
+    );
   }
 
   return (
-    <>
-      <StyledContainer>
-        <StyledImage
-          src={currentSite.currentPicture[0]}
-          alt={currentSite.adress}
-          fill
-        />
-      </StyledContainer>
-      <p>
-        <StyledLink href="/singlesite">
-          <Image
-            src={"/profan--rosybrown.svg"}
-            alt="Profan-Logo"
-            height={50}
-            width={50}
+    <StyledContainer>
+      <Swiper
+        className="mySwiper swiper-h"
+        spaceBetween={0}
+        slidesPerView={1}
+        centeredSlides={true}
+        pagination={{
+          clickable: true,
+        }}
+        modules={[Pagination]}
+        onSlideChange={() => console.log("slide change")}
+        onSwiper={(swiper) => console.log(swiper)}
```

Comment:
please don`t leave the logs in your PRs. I guess there should be some functionality. :)